spec/packaging/ibm_cloud_pak/qualification-yaml.md

```diff
@@ -5,20 +5,20 @@ The qualification.yaml file includes the following sections:
 * [catalog](#catalog-section)
 
 ## Qualification Section
-The qualification section describes attributes related to the IBM Cloud Pak status.
+The qualification section describes attributes related to the IBM&reg; Cloud Pak status.
 
 Key name:  `qualification`
 
 **Fully Qualified Attribute Name**|**Type**|**Description/Specification**
 -----|-----|-----
-qualification.levelName|String|Value must be: `ibm-cloud-pak`
-qualification.levelDescription|String|Value must be: `IBM Cloud Pak`
+qualification.levelName|String|One of two pre-defined levels: `ibm-cloud-pak`, `certified-ibm-cloud-pak`, `ibm-solution-pak`, `certified-ibm-solution-pak`
```

Comment:
on of two predefined levels - should this be 4? or by pre-defined level do you mean cloud-pak verses solution-pak?